packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmailToken/template.success.tsx

```diff
@@ -38,6 +45,23 @@ const Success = () => {
           />
         </Text>
       )}
+      <Padding top={1}>
+        <Button
+          data-e2e='verifyEmail'
+          fullwidth
+          height='48px'
+          nature='light'
+          onClick={returnToApp}
+          style={{ marginBottom: '5px', marginTop: '32px' }}
```

Comment:
you could remove this style by adding the the 5px and 32px to the Padding component above. Also, you could swap this button for a constellation button!